boards/nxp/imx93_evk/doc/index.rst

```diff
@@ -71,22 +71,42 @@ hardware features:
 | ENET      | on-chip    | ethernet port                       |
 +-----------+------------+-------------------------------------+
 
+The Zephyr imx93_evk board Cortex-M33 configuration supports the following
+hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | interrupt controller                |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | clock_control                       |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | GPIO                                |
++-----------+------------+-------------------------------------+
+
 Devices
 ========
 System Clock
 ------------
 
 This board configuration uses a system clock frequency of 24 MHz.
 Cortex-A55 Core runs up to 1.7 GHz.
+Cortex-M33 Core runs up to 200MHz in which SYSTICK runs on same frequency.
 
 Serial Port
 -----------
 
 This board configuration uses a single serial communication channel with the
-CPU's UART4.
+CPU's UART4 for A55 core, and UART2 for M33 core.
```

Comment:
Actually A55 Zephyr also use LPUART2 as debug console, please fix it.

Reply:
Should I add one more dts patch to change A55 uart for debugging, keeping same with Linux BSP?
Thanks.

Reply:
No, let's keep zephyr to use LPUART2 both for A-Core and M-Core, so that it can runs together with Linux. Let's just update this document correctly.

Reply:
Okay for me. But I'm not sure if there is case in the future that both A core and M core runs zephyr at same time.